package-lock.json

```diff
@@ -2975,14 +2975,20 @@
       }
     },
     "node_modules/caniuse-lite": {
-      "version": "1.0.30001312",
```

Comment:
@theodab Is this change necessary in this PR?

Reply:
Ah, no, that was an accident.

Reply:
The difference between "npm install" and "npm ci" is that "install" will update the package lock and find new versions that satisfy the requirements in package.json.  "ci" will only install exactly what is in the lock file, for consistent reproducibility in CI systems.